.github/dependabot.yml

```diff
@@ -4,14 +4,16 @@
 version: 2
 updates:
   # Enable version updates for npm
-  - package-ecosystem: "npm"
+  - package-ecosystem: 'npm'
     # Look for `package.json` and `lock` files in the `root` directory
-    directory: "/"
+    directory: '/'
     # Check the npm registry for updates every day (weekdays)
     schedule:
-      interval: "daily"
+      interval: 'daily'
+    versioning-strategy: 'increase'
```

Comment:
Can you explain why you're adding this line as part of this PR? Also it's generally polite to follow the conventions of the library you're contributing to, especially as a first time contributor, and avoid rewriting an entire file's style to match your own preferences.

Reply:
Apologies. I didn't even realise it has changed the ticks. This change is kind of outside the scope of the rest of the PR so let me put in it in a separate PR where I'll explaining the reasoning and it can be reviewed separately.

Reply:
I'm not strict on double vs single quotes -- even prefer single quotes when I'm starting fresh. This file was likely copied from a dependabot template when dependabot was first released to the public. Just generally discourage drive-by style changes from new contributors. Thanks for understanding :)

If you're motivated to open a PR to enforce style guidelines I won't stop you, but it's not a high priority since this repo has been in maintenance mode for a long time.